modules/ROOT/Edits/configure-partner.adoc

```diff
@@ -0,0 +1,17 @@
+= Configure and Manage Partners
+
+Use the *Partners* page to create and manage partners. From this page, you can:
+
+* xref:create-partner.adoc[Create a new partner]
+* xref:partner-manager-identifiers.adoc[Add the EDI and AS2 identifiers] used by your partner.
+* Add digital xref:Certificates.adoc[certificates] associated with your partner.
+* xref:create-endpoint.adoc[Create] and manage source and target endpoints owned by the partner.
```

Comment:
Remove bullets

Reply:
Not sure why you want me to remove bullets.